src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.cs

```diff
@@ -445,7 +445,7 @@ public void TestCapacityReservation()
             TestRunner.RunTestScript("Test-CapacityReservation");
         }
 
-        [Fact(Skip = "Cannot run on Github Platform due to ACL API")]
+        [Fact]
```

Comment:
How is the test not changing and remaining useful if the code is changing substantially?